index.d.ts

```diff
@@ -80,6 +80,10 @@ declare module 'react-helmet-async' {
     context?: {};
   }
 
+  export class HelmetData {
```

Comment:
For some reason I assumed that the types were auto generated, which makes no sense because the lib is JS not TS 🙈